source/index.html.md.erb

```diff
@@ -15913,20 +15980,63 @@ The FormatTables Object is retrieved via the `GET /serverinfo` operation and con
 **numberFormat** | array of strings | Possible values:<ul><li>none</li><li>NUMBER</li><li>CURRENCY</li><li>PERCENT</li></ul>
 **textWrap** | array of strings | Possible values:<ul><li>none</li><li>on</li></ul>
 
-## FontFamily Object
+<aside class="notice">Indexes and their values are guaranteed never to change or be removed for a given major API version.  However, new values could potentially be added to the ends of lookup tables.<br/><br/>Because of this possibility, your code should handle the case where a cell might contain a format index value greater than the size of a lookup table your app has loaded.  Your application should check for that case and reload the format tables if necessary.</aside>
+
+### FontFamily Object
+
+> Example FontFamily Object:
+
+```json
+{
+	"fontFamily": [{
+		"name": "Arial",
+		"traits": [
+			"sans-serif"
+		]
+	}]
+}
+```
 
      |     |     |
 -----|-----|-----|
 **name** | string | Name of the font family (for example, "Arial")
 **traits** | array | Platform-independent traits of the font family. One of the following values:<ul><li>serif</li><li>sans-serif</li>
 
-## Currency Object
+### Currency Object
+
+> Example Currency Object:
+
+```json
+{
+	"currency": [{
+		"code": "USD",
+		"symbol": "$"
+	}]
+}
+```
 
      |     |     |
 -----|-----|-----|
-**code** | string | The ISO 4217 currency code (for example, "EUR")
+**code** | string | The [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) currency code (for example, "EUR")
 **symbol** | string | The currency symbol (for example, "€")
 
+## Get Formatting
+
+Use the "include=format" query-string parameter on API operations that return detailed objects, such as `GET /sheets/{sheetId}` or `GET sheets/{sheetId}/rows/{rowId}`. If there is formatting other than default settings, the return includes a **format** property.
+
+## Get Conditional Formatting
+
+Use the "include=format" query-string parameter. If an object has conditional formatting, the **format** property returned will have a **conditionalFormat** value.
+
+## Set Formatting for a Row or Column
+
+Setting the format of a row or column through the API simply sets the baseline format for new cells in that row or column.  It does not affect existing cells.
```

Comment:
Have you tested setting formatting for a row or col? (I'm unclear on what's possible)

Reply:
Yes. It won't change any cells that already have values, but if you change either a row or column to italic (for instance), then any newly inputted text will be italic. Even a picklist will type in as italic, and then revert to the picklist formatting when you hit enter.